README.md

```diff
@@ -10,6 +10,12 @@ RRRSpec is used in production as a CI service, running 60+ RSpec processes
 concurrently, and it undergoes those failures, which include lots of `rb_bug`s,
 assertion errors, and segmentation faults.
 
+## MAINTENANCE NOTICE
+
+As of Feb 2019, we, Cookpad, are not using RRRSpec internally. We may ocassionaly review and accept PRs, but note that this software is not under active maintainance.
+
+(Background: The circumstances has been changed since 2013. Modern IaaS providers such as AWS now provides a machine with many vCPUs under the cheap pricing. Also, Cookpad is working to split their largest Rails monolith application ([blog post :jp:](https://techlife.cookpad.com/entry/2018-odaiba-strategy)). Thus, we decided to stop operating RRRSpec cluster.)
```

Comment:
nit: provides -> provide

Reply:
the cheap pricing -> cheap pricing (remove the the)

Reply:
a machine -> machines